src/go/rpk/pkg/cli/cmd/wasm/remove.go

```diff
@@ -61,16 +62,13 @@ func remove(
 	}
 	// create empty message, the remove command doesn't need
 	// information on message, just a key value
-	var emptyMessage []byte
 	header := createHeader("remove")
+	// create id from filename
+	id := xxhash.Sum64([]byte(name))
```

Comment:
This logic can be inside of CreateCoprocessorMessage

Reply:
make senses